sdk/package.json

```diff
@@ -9,8 +9,8 @@
     "update-relay": "ts-node src/scripts/relay-retarget.ts",
     "build": "tsc -p tsconfig.json",
     "lint": "eslint .",
-    "format": "prettier --write {examples,src,test}",
-    "format:check": "prettier --check {examples,src,test}"
+    "format": "prettier --write examples src test",
+    "format:check": "prettier --check examples src test"
```

Comment:
Was this not working before?

Reply:
I don't know if it's a mac vs linux things but the previous syntax does not work for me

either works for me:
- `prettier --write examples src test`
- `prettier --write \"{examples,src,test}/**/*.ts\"`